web/viewer.css

```diff
@@ -682,19 +686,19 @@ html[dir='rtl'] .dropdownToolbarButton:after {
 }
 
 .dropdownToolbarButton > select {
-  min-width: 140px;
-  font-size: 12px;
+  width: 100%;
```

Comment:
Similar to the comment above, please revert this since it leads to an incorrect/inconsistent width until the `select` has been localized.